nixos/roles/devhost/default.nix

```diff
@@ -181,6 +190,18 @@ in
         fi
       '';
 
+      # BBB can be removed after a grace period
+      flyingcircus.activationScripts.devhost-migrate-ip-config = ''
+        set -x
+        cd /etc/devhost
```

Comment:
Is it ok to ignore the old `/etc/devserver` here?

Reply:
Yeah, this has been rolled out already and I'm assuming the other script runs first in any case. Also, even if it doesn't it's working in a converging manner, so ...